lib/istio/tracing.rb

```diff
@@ -2,3 +2,14 @@
 require "istio/tracing/header"
 require "istio/tracing/http"
 require "istio/tracing/rack"
+
+begin
+  require 'rails/railtie'
+  rails_exists = true
+rescue LoadError
+  rails_exists = false
+end
+
+if rails_exists
+  require 'istio/tracing/railtie'
+end
```

Comment:
Do you plan to add more lines in this if block? 

If it is just for requiring `'istio/tracing/railtie'`,  we can omit `rails_exists` variable. 

```ruby
begin
  require 'rails/railtie'
  require 'istio/tracing/railtie'
rescue LoadError
end
```

What do you think?

Reply:
Fixed it.